cs-ilp-codec-framework/Org/Interledger/Encoding/Asn/Serializers/Oer/OerLengthSerializer.cs

```diff
@@ -71,12 +71,15 @@ private static ulong ToUlong(byte[] bytes)
         {
             // because length of bytes is not always 8.
             // e.g. 3 bytes, it must be 8 bytes to be converted to ulong.
-            Array.Reverse(bytes);
-            if (bytes.Length < 8)
+            if (BitConverter.IsLittleEndian)
```

Comment:
`BigInteger` returns and treats bytes as little-endian, but how `BitConverter` treats bytes is up to the architecture.